SignalMessaging/categories/UIColor+OWS.m

```diff
@@ -27,6 +27,11 @@ + (UIColor *)ows_blackColor
     return [UIColor colorWithRed:8.f / 255.f green:10.f / 255.f blue:0. / 255.f alpha:1.f];
 }
 
++ (UIColor *)ows_blackIconColor
```

Comment:
Is `ows_blackIconColor` the best name for something that isn't black?

Reply:
<img src="https://media.giphy.com/media/xT5LMVNklFTi4CnieI/giphy.gif">

Reply:
"dark white"?